cmd/lsjson/lsjson.go

```diff
@@ -106,6 +113,27 @@ can be processed line by line as each item is written one to a line.
 					if !noModTime {
 						item.ModTime = Timestamp(entry.ModTime())
 					}
+					if showEncrypted {
+						fsInfo, configName, _, err := fs.ParseRemote(args[0])
```

Comment:
Making the `cipher` is expensive - this bit of code (from line 117 to 128 the switch) should be removed from the Walk function so you don't do it for every item. I'd probably put it after `fsrc := cmd.NewFsSrc(args)`